src/jit/morph.cpp

```diff
@@ -13108,7 +13108,7 @@ GenTree* Compiler::fgMorphSmpOp(GenTree* tree, MorphAddrContext* mac)
                         // The type of the new GT_NEG node cannot just be op1->TypeGet().
                         // Otherwise we may sign-extend incorrectly in cases where the GT_NEG
                         // node ends up feeding directly a cast, for example in
-                        // GT_CAST<ubyte>(GT_MUL(-1, s_1.ubyte)
+                        // GT_CAST<ubyte>(GT_MUL(-1, s_1.ubyte))
```

Comment:
You've gotta love how the formatting tools will randomly start complaining about something that's been there forever.

Reply:
I messed up this comment in #18837 but there was no reason to fix it there and run CI again. I forgot to fix it in the first commit on this PR, but clang was unhappy about the switch missing the default case, so I figured I would fix the comment together with that.

So not the formatting... this time.